.github/workflows/cla-assistant.yaml

```diff
@@ -0,0 +1,47 @@
+name: "CLA Assistant"
+on:
+  issue_comment:
+    types: [created]
```

Comment:
Unfortunately from GitHub's perspective [PR comments ARE issue comments](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#pull_request_comment-use-issue_comment). Hmm I could make it do a noop for "true" issue comments so it exits fast. Since this is a public repo I believe this should be no cost.